src/gbd_mapping_generator/cause_builder.py

```diff
@@ -21,16 +21,16 @@ def get_base_types():
         ("name", "str"),
         ("kind", "str"),
         ("gbd_id", ID_TYPES.C_ID),
-        ("me_id", f"Union[{ID_TYPES.ME_ID}, Unknown]"),
+        ("me_id", f"{ID_TYPES.ME_ID} | Unknown"),
```

Comment:
I wonder if changes like this are going to break the builders. Did you actually test that?

Reply:
Yeah I mentioned in the description, but to go into a bit more detail - I built all of the templates, ran isort (the templates don't adhere to linting), and did a git diff.

Reply:
I'm gonna go ahead and fix the isort issues actually since I need to add from __future__ import annotations anyway

Reply:
Out of scope, but do we have automated test coverage of the builder code? If not, we should create a JIRA ticket to create that.